### PR TITLE
fix errors during test-lint

### PR DESCRIPTION
**Pull request by adamwawrzynski3mdeb**
Signed-off-by: Adam Wawrzyński <adam.wawrzynski@3mdeb.com>

### CODE REVIEW

**Comment by adamwawrzynski3mdeb** [comment]
@pietrushnic What should be copyrights? 3mdeb, pcengines or other?

File: src/mainboard/pcengines/apu2/romstage.c [hunk 1]
```diff
@@ -183,11 +183,11 @@ static void early_lpc_init(void)
 	//
 	// Configure output disabled, value low, pull up/down disabled
 	//
-#if CONFIG_BOARD_PCENGINES_APU5
+#if iS_ENABLED(CONFIG_BOARD_PCENGINES_APU5)
```
Comment (miczyg1) [inline src/mainboard/pcengines/apu2/romstage.c]: @adamwawrzynski3mdeb small case typo

File: src/mainboard/pcengines/apu2/Kconfig [hunk 2]
```diff
@@ -33,7 +33,7 @@ config BOARD_SPECIFIC_OPTIONS # dummy
 	select GENERIC_SPD_BIN
 	select SPI_FLASH
 	select SPI_FLASH_WINBOND
-	select TPM
+	select TPM1
```
Comment (pietrushnic) [inline src/mainboard/pcengines/apu2/Kconfig]: @miczyg1 are we sure this is correct?

Comment (adamwawrzynski3mdeb) [reply src/mainboard/pcengines/apu2/Kconfig]: @miczyg1 https://github.com/pcengines/coreboot/blob/014d812ebdea72c7046de1c2e2022939439e999a/src/security/tpm/Kconfig#L22

Comment (miczyg1) [reply src/mainboard/pcengines/apu2/Kconfig]: @pietrushnic question is why we had TPM selected in the first place. If we want to keep things as it were before, we should keep it. Either way it is selected by two options below:
```
MAINBOARD_HAS_LPC_TPM
MAINBOARD_HAS_TPM1
```
This select does not affect anything:
https://github.com/pcengines/coreboot/blob/014d812ebdea72c7046de1c2e2022939439e999a/src/security/tpm/Kconfig#L23

Comment (pietrushnic) [reply src/mainboard/pcengines/apu2/Kconfig]: @miczyg1 yes, but our module is SLB9665 TPM2.0

Comment (miczyg1) [reply src/mainboard/pcengines/apu2/Kconfig]: @pietrushnic Yes and probably we Will need to push patches to enable it. For now MAINBOARD_HAS_TPM1 has been added in mainline due to some bigger change to TPMs code. If we want to support mentioned module, we have to verify firstly what we need to enable in the code.

Comment (pietrushnic) [reply src/mainboard/pcengines/apu2/Kconfig]: @miczyg1 agree. This is on my TODO right after IOMMU.

**Pull request merged**